torchao/quantization/quantize_/workflows/intx/intx_opaque_tensor.py

```diff
@@ -134,15 +108,24 @@ def from_intx_unpacked_to_int8_tensor(
         tensor: IntxUnpackedToInt8Tensor,
         *,
         bias: Optional[torch.Tensor] = None,
-        compute_target: Union[ComputeTarget, str] = ComputeTarget.TORCHAO_AUTO,
+        intx_packing_format: Union[
```

Comment:
nit: this can just be `IntxPackingFormat` since it's already inherits from both str and Enum